lib/pure/uri.nim

```diff
@@ -162,21 +162,25 @@ func encodeQuery*(query: openArray[(string, string)], usePlus = true,
 
 iterator decodeQuery*(data: string): tuple[key, value: TaintedString] =
   ## Reads and decodes query string ``data`` and yields the (key, value) pairs the
-  ## data consists of.
+  ## data consists of. If compiled with -d:nimLegacyParseQueryStrict, an error
+  ## is raised when there is an unencoded ``=`` character in a decoded value,
+  ## which was the behavior in Nim < 1.6
```

Comment:
```nim
  ## Reads and decodes query string `data` and yields the `(key, value)` pairs the
  ## data consists of. If compiled with `-d:nimLegacyParseQueryStrict`, an error
  ## is raised when there is an unencoded `=` character in a decoded value,
  ## which was the behavior in Nim < 1.5.1
```

(changed a few things, including https://nim-lang.github.io/Nim/contributing.html
> In nim sources, prefer single backticks

)

Reply:
done

Reply:
@mildred thanks; in future, please click "resolve conversation" when something is resolved (unless it's a pushback); thanks!